nix/pkgs/haskell/materialized-unix/.plan.nix/Win32-network.nix

```diff
@@ -108,4 +108,4 @@
           };
         };
       };
-    } // rec { src = (pkgs.lib).mkDefault .././.source-repository-packages/18; }
\ No newline at end of file
+    } // rec { src = (pkgs.lib).mkDefault .././.source-repository-packages/19; }
```

Comment:
I've added a new pinned dependency (`flat`) on a GitHub repo (rather than hackage). I think this is why `.nix` files needed to be updated. @michaelpj is everything fine here?